elao.app/.manala/ansible/inventories/system.yaml.tmpl

```diff
@@ -398,6 +398,9 @@ system:
         {{- if .mysql.version }}
         manala_mysql_configs_dir: /etc/mysql/mysql.conf.d
         {{- end }}
+        {{- if gt (.mariadb.version|float64) 10.4 }}
```

Comment:
hm, unfortunately, .mysql.version and .mariadb.version are not mutually exclusives... I wonder if it would be better with an "else if" or not...

Reply:
done